Common/src/containers/CLookUpTable.cpp

```diff
@@ -574,7 +573,13 @@ unsigned long CLookUpTable::LookUp_XY(const string& val_name_var, su2double* val
                                       su2double val_CV2, unsigned long i_level) {
   unsigned long exit_code = 1;
 
-  if (val_name_var.compare("NULL") == 0) {
+  if (val_name_var.compare("NULL") == 0 ||
+      val_name_var.compare("Null") == 0 ||
+      val_name_var.compare("null") == 0 ||
+      val_name_var.compare("ZERO") == 0 ||
+      val_name_var.compare("Zero") == 0 ||
+      val_name_var.compare("zero") == 0 )
```

Comment:
I would put this in a function

Reply:
OK, I am using an inline function if (noSource(val_name_var)) now.